mesh/node.c

```diff
@@ -1426,6 +1785,8 @@ static void get_managed_objects_cb(struct l_dbus_message *msg, void *user_data)
 	struct managed_obj_request *req = user_data;
 	const char *path;
 	struct mesh_node *node = NULL;
+	struct l_queue *net_keys;
```

Comment:
`mesh/node.c:1949:5: error: ‘net_keys’ may be used uninitialized in this function [-Werror=maybe-uninitialized]`